### PR TITLE
Dynamic index page 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
### Description 
Changes made so a visitor and a logged in user sees different content on index page.
- Extract index content to a partial 
- Add index content for logged in user in a partial 
- Add render method on index page for user and visitor 

### Screenshot 
screen shot 2018-02-07 at 15 49 53

screen shot 2018-02-07 at 15 50 22

### CODE REVIEW

File: features/user_can_see_relevant_info_when_logged_in.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  As a user,
+  In order to know that I am logged in,
+  I would like to see a welcome message and my profile info
+
+  Background:
+    Given the following user exists
+       | email                | password       | password_confirmation     |
+       | lisa@gmail.com       | heythere123    | heythere123               |
+
+    When I visit the index page
```
Comment (lcegit) [inline features/user_can_see_relevant_info_when_logged_in.feature:11]: We could add an 'I should see' 'A social family platform' for the visitor before logging in.

File: features/user_can_see_relevant_info_when_logged_in.feature [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    And I click "Log in"
+
+  Scenario:
+    When I fill in "Email" with "lisa@gmail.com"
+    And I fill in "Password" with "heythere123"
+    And I click "Log in" button
+    Then I should be redirected to the "index" page
+    And I should see "Welcome"
```
Comment (lcegit) [inline features/user_can_see_relevant_info_when_logged_in.feature]: Should we also add what is unique / the user's name for the expected greeting?

**Review by lcegit — COMMENTED** [review]
Two minor comments, otherwise it looks great!

**Review by NegativeKarma — APPROVED** [review]
LGTM

**Review by lcegit — APPROVED** [review]
Looks great!

**Pull request merged**